### DISCUSSION TITLE
问题回馈

### DISCUSSION THREAD

**Discussion by EATABIGMUSHROOM** [STranslate/STranslate#239] [Q&A]
我发现在打开“划词翻译”功能后，STranslate软件会置顶，而后卡住某些界面无法显示出来（任务栏鼠标悬停可预览，但点击无法弹出），目前碰到被卡住的有WPS和edge，系统是23H2，win11；在两台电脑都有这样的问题，软件版本均是最新STranslate 1.2.6.1001

**Comment by ZGGSONG** [comment]
> 我发现在打开“划词翻译”功能后，STranslate软件会置顶，而后卡住某些界面无法显示出来（任务栏鼠标悬停可预览，但点击无法弹出），目前碰到被卡住的有WPS和edge，系统是23H2，win11；在两台电脑都有这样的问题，软件版本均是最新STranslate 1.2.6.1001
> 
> 

我手头暂时没有win11设备，回头测试一下啥原因
